### PR TITLE
Use appropriate workflow url for workflow_dispatch event trigger

### PR DESCRIPTION
**Pull request by studuffy-moia**
without this change, for runs from the `workflow_dispatch` trigger, the url as is can point to a misleading run. it will always direct to the latest run on the branch in context, but obviously for anything that isn't the most recent run any more this will be incorrect.  so in this case, just use the run id instead.

### CODE REVIEW

**Comment by cbrgm** [comment]
LGTM, thanks for contributing @studuffy-moia ! ❤️

**Comment by cbrgm** [comment]
Waiting for one more review, @jvm986 maybe?

**Review by jvm986 — APPROVED** [review]
looks good to me

**Pull request merged**
